src/containers/Storage/DiskStateProgressBar/DiskStateProgressBar.tsx

```diff
@@ -63,13 +59,7 @@ function DiskStateProgressBar({
             aria-valuemax={100}
             aria-valuenow={diskAllocatedPercent}
         >
-            {href ? (
-                <InternalLink to={href} className={b('link')}>
-                    {renderAllocatedPercent()}
-                </InternalLink>
-            ) : (
-                renderAllocatedPercent()
-            )}
+            {renderAllocatedPercent()}
```

Comment:
This makes this component a single purpose, only displaying a disk state. It can be wrapped in a link if needed. Simultaneously, this fixes the issue with media type label blocking the link active area